tensorflow/c/experimental/filesystem/plugins/gcs/BUILD

```diff
@@ -42,3 +42,26 @@ cc_library(
         "//tensorflow/c:env",
     ],
 )
+
+tf_cc_test(
+    name = "gcs_filesystem_test",
+    srcs = [
+        "gcs_filesystem.cc",
+        "gcs_filesystem_test.cc",
+    ],
+    local_defines = ["TF_GCS_FILESYSTEM_TEST"],
+    tags = [
+        "manual",
+    ],
+    deps = [
+        "gcs_helper",
+        "//tensorflow/c:tf_status",
+        "//tensorflow/c/experimental/filesystem:filesystem_interface",
+        "@com_github_googlecloudplatform_google_cloud_cpp//:storage_client",
+        "@com_google_absl//absl/strings",
+    ] + [
```

Comment:
No need to concatenate two lists, let's merge them together

Reply:
This list is seperated because one list is dependencies of `gcs_filesystem` and the second is the dependencies of `gcs_filesystem_test`